lux/test/unit/lux/llm/together_ai_test.exs

```diff
@@ -0,0 +1,334 @@
+                 parameters: %{
+                   type: "object",
+                   properties: %{
+                     "value" => %{
+                       type: "string",
+                       description: "Test value"
+                     },
+                     "amount" => %{
+                       type: "float",
+                       description: "Test amount"
+                     }
+                   }
+                 }
+               }
+             } = function
+    end
+
+    test "converts a prism to a Together AI function" do
+      prism = TestPrism.view()
+
+      function = TogetherAI.tool_to_function(prism)
+
+      assert %{
+               type: "function",
+               function: %{
+                 name: "Lux_LLM_TogetherAITest_TestPrism",
+                 description: "A test prism",
+                 parameters: %{
+                   type: :object,
+                   properties: %{
+                     value: %{
+                       type: :string
+                     }
+                   }
+                 }
+               }
+             } = function
+    end
+
+    test "converts a lens to a Together AI function" do
+      lens =
+        Lux.Lens.new(
+          name: "WeatherAPI",
+          description: "Gets weather data",
+          schema: %{
+            type: "object",
+            properties: %{
+              location: %{
+                type: "string",
+                description: "City name"
+              },
+              units: %{
+                type: "string",
+                description: "Temperature units"
+              }
+            }
+          }
+        )
+
+      function = TogetherAI.tool_to_function(lens)
+
+      assert %{
+               type: "function",
+               function: %{
+                 name: "WeatherAPI",
+                 description: "Gets weather data",
+                 parameters: %{
+                   type: "object",
+                   properties: %{
+                     location: %{
+                       type: "string",
+                       description: "City name"
+                     },
+                     units: %{
+                       type: "string",
+                       description: "Temperature units"
+                     }
+                   }
+                 }
+               }
+             } = function
+    end
+  end
+
+  describe "call/3" do
+    test "makes correct API call with tools" do
+      config = %{
+        api_key: "test_key",
+        model: "mistralai/Mistral-7B-Instruct-v0.2"
+      }
+
+      beam =
+        Lux.Beam.new(
+          name: "TestBeam",
+          description: "A test beam",
+          input_schema: %{
+            type: "object",
+            properties: %{
+              "value" => %{
+                type: "string",
+                description: "Test value"
+              }
+            }
+          }
+        )
+
+      Req.Test.expect(TogetherAI, fn conn ->
```

Comment:
From how this and a few other tests are failing to pass locally and in CI, it seems like the mock is not working and `TogetherAI.call` is actually trying to make a call to the internet.